client/components/Profile.jsx

```diff
@@ -25,7 +29,7 @@ class Profile extends React.Component {
   }
 
   render () {
-    const {username, currentPassword, newPassword, confirm, showMatch, match} = this.state
+    const {username, email, cohort, profilePic, bio, currentPassword, newPassword, confirm, showMatch, match} = this.state
```

Comment:
Let's break this long single line into at least 2 lines so we don't exceed 80 characters. If it is shorter than 80 characters already, you don't need to change it.